themes/gatsby-starter-blog-theme/package.json

```diff
@@ -9,7 +9,7 @@
   },
   "dependencies": {
     "gatsby": "^2.13.23",
-    "gatsby-theme-blog": "^1.0.0",
+    "gatsby-theme-blog": "^0.0.13",
```

Comment:
We update `package.json` and lock files automatically via Renovate so this is unnecessary as well